test/unittest/CMakeLists.txt

```diff
@@ -101,6 +101,31 @@ if(is_dpcpp)
   )
 endif()
 
+if(is_dpcpp AND ${TUNING_TARGET} STREQUAL "DEFAULT")
+  if (${DPCPP_SYCL_TARGET} STREQUAL "nvptx64-nvidia-cuda")
+    set(DEFAULT_SKIP
+      ${PORTBLAS_UNITTEST}/blas1/blas1_iamax_test.cpp
+      ${PORTBLAS_UNITTEST}/blas1/blas1_iamin_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_tbsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_tpsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_trsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas3/blas3_trsm_test.cpp
+      )
+    message(WARNING "Targetting NVIDIA hardware with DEFAULT TUNING_TARGET.
+    Disabling tests for following operators: iamax, iamin, trsv, tbsv, tpsv, trsm.")
+  elseif(${DPCPP_SYCL_TARGET} STREQUAL "amdgcn-amd-amdhsa")
+    set(DEFAULT_SKIP
+      ${PORTBLAS_UNITTEST}/blas1/blas1_iamax_test.cpp
+      ${PORTBLAS_UNITTEST}/blas1/blas1_iamin_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_tbsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_tpsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_trsv_test.cpp
+      )
+    message(WARNING "Targetting AMD hardware with DEFAULT TUNING_TARGET.
+    Disabling tests for following operators: iamax, iamin, tbsv, tpsv, trsv.")
+  endif()
```

Comment:
Should these be more general - i.e. also catch targets like `nvidia_gpu_sm_80` and `amd_gpu_gfx90a`?